cmd/leanix-k8s-connector/main.go

```diff
@@ -43,16 +43,20 @@ func main() {
 	if err != nil {
 		logger.Error("Error occurred when authenticating.", err)
 		logger.Info("Failed to authenticate. Terminating..")
+		return
 	}
 	if viper.GetBool(utils.IrisFlag) {
 		logger.Info("Enabled Iris")
 		runId := iris.GenerateRunId()
 		irisScanner := iris.NewScanner(
 			"Iris Integration",
-			viper.GetString(utils.IntegrationAPIFqdnFlag), runId,
+			viper.GetString(utils.IntegrationAPIFqdnFlag),
```

Comment:
Since we no longer use integration API, can we rename ´IntegrationAPIFqdnFlag´to ´host´ or something else to make it clearer that this is the instance/region?

Reply:
Yes we can do that. There are just some things to it:
1. Align with UI to show the right parameter names in the examples when setting up the k8s config
2. Backwards compatibility might be a thing so maybe we have to work with two parameters in between

Reply:
Are there any customers using the non-iris version?

Reply:
Yes, Ragu just came up with a customer where we have to implement a quick fix for integration api version 😓

Reply:
I added some new parameters and some fallback. Please check.